oak_functions/client/src/bin/oak_functions_client.rs

```diff
@@ -17,17 +17,10 @@
 //! Sends a gRPC request to the weather lookup application and checks that the response has the
 //! correct format.
 
-pub mod proto {
```

Comment:
Why is this file moved? Wouldn't `oak_functions/client/src/main.rs` work?

Reply:
I thought I tried that first and it didn't work for some reason, but now trying again it actually works :thinking: moved back!